src/cljs/main/broadfcui/common/style.cljs

```diff
@@ -95,6 +95,13 @@
        (cons [:option {:value -1 :disabled true} (first placeholder)] option-elements)
        option-elements)]))
 
+(defn create-identity-select-name [props options & placeholder]
+  (let [option-elements (map (fn [opt] [:option {:value opt} opt]) options)]
+    [:select (utils/deep-merge {:style select-style} props)
+     (if-not (empty? placeholder)
+       (cons [:option {:value -1 :disabled true} (first placeholder)] option-elements)
+       option-elements)]))
```

Comment:
What happens if you provide 4 arguments? My guess is that the 4th would be ignored. If that is true then I'd prefer to see the longer multi-arity defn form so the compiler will warn when there are too many args.